x-pack/plugins/security_solution/public/management/components/endpoint_response_actions_list/components/hooks.tsx

```diff
@@ -213,12 +264,16 @@ export const useActionsLogFilter = (
   const numFilters = useMemo(() => items.filter((item) => item.checked !== 'on').length, [items]);
 
   return {
+    areHostsSelectedOnMount,
+    isLoading: isHostsFilter && isFetching,
```

Comment:
I would avoid set loading to true when is fetching as the `keepPreviousData` is globally enabled. It will avoid having this blank effect when the user types in the host name.

Reply:
I see what you mean. Without the loading state, the options get updated suddenly sometimes. See here
![hosts-filter-non-loading](https://user-images.githubusercontent.com/1849116/190471463-83269efa-b451-40a0-a54a-a724b33f6196.gif).

We do need the loader as we are only showing 50 options but there might be more (unseen) options that the user wants to search (and re-fetch the list).

![hosts-filter-loading](https://user-images.githubusercontent.com/1849116/190471781-dddb3635-91b6-4600-89b5-f047e13d1647.gif)